### ISSUE TITLE
script IPChecker.sh - script doesn't act as expected

### ISSUE THREAD

**Issue by littlejeem** [littlejeem/monitor_scripts#11]
script runs.

Test condition was to stop OpenVPN, leave transmission-daemon running and run the script.

Result was as follows

```bash
script STARTED - 22.25-17_06_2020
/home/pi/bin/monitor_scripts
I would source /home/pi/bin/monitor_scripts/config.sh in real world
VPN is DOWN
Attempting to stop transmission-daemon
```

Two pushovers received

1:
```VPN is DOWN```
followed by
2:
```failed to successfully 'stop' transmission-daemon, urgent attention required```

The expected behaviour was for the first pushover and for the script to stop transmission-daemon.

**Comment by littlejeem** [comment]
script now runs as expected but exits with
```bash
rm: cannot remove '/tmp/IPChecker.lock': No such file or directory
```

**Comment by littlejeem** [comment]
> script now runs as expected but exits with
> 
> ```shell
> rm: cannot remove '/tmp/IPChecker.lock': No such file or directory
> ```

fixed by 8872d29ba9423ab750cae554d3a02d9dc70e6571
